services/web/server/src/simcore_service_webserver/users/_preferences_models.py

```diff
@@ -95,6 +95,11 @@ class TelemetryLowDiskSpaceWarningThresholdFrontendUserPreference(
     value: int = 5  # in gigabytes
 
 
+class BillingCenterUsageColumnOrderFrontendUserPreference(FrontendUserPreference):
+    preference_identifier: PreferenceIdentifier = "billingCenterUsageColumnOrder"
+    value: list[int] | None = None
```

Comment:
Q: just to make sure, you can make this one return by default an empty list. You sure you don't need that and prefer the None.